config/config.inc.php

```diff
@@ -52,11 +52,11 @@
 $config['folders']['tmp'] = 'data/tmp';
 $config['folders']['data'] = 'data';
 
-// WEDDING SETTINGS
-$config['is_wedding'] = false;
-$config['wedding']['groom'] = 'Name 1';
-$config['wedding']['bride'] = 'Name 2';
-$config['wedding']['symbol'] = 'fa-heart-o';
+// Event SETTINGS
+$config['is_event'] = false;
```

Comment:
I think this is a breaking change and that would mean we have to make a major version bump. But I'm not 100% sure.

Reply:
We'd only need to mention to remove my.config.inc.php.

We could also think about adding the option what kind of reset is done pressing "reset" in admin panel (checkboxes: 1. Delete all personal data 2. Delete personal config) , by default enable both.

Reply:
2.1 isn't released yet, push it with it?

Reply:
That would probably the easiest thing to do.